### PR TITLE
Flame thrower fix

### PR DESCRIPTION
**Pull request by RebeccaStankus**
TEST PLAN

Go to staging 1 and equip the flamethrower in desktop. The animation should look realistic with avatar arm outstretched and flamethrower in hand pointing forward. Check that both hands have the correct animation and gun position.

https://highfidelity.fogbugz.com/f/cases/18880/Equipping-items-in-the-Spot-uses-wrong-positioning

### CODE REVIEW

**Comment by AndySeattle** [comment]
@RebeccaStankus let me know when this is QA ready

**Comment by AndySeattle** [comment]
We checked this in the spot, so when we get a code review on this, we could test again when you make as QA ready.

File: DomainContent/Hub/RainbowThrower/scripts/rainbowThrowerGunSpawner.js [hunk 1]
```diff
@@ -28,6 +27,7 @@
         preload: function(entityID) {
             _this.entityID = entityID;
             _this.position = Entities.getEntityProperties(_this.entityID, 'position').position;
```
Comment (misslivirose) [inline DomainContent/Hub/RainbowThrower/scripts/rainbowThrowerGunSpawner.js]: These should get consolidated into a single `getEntityProperties` call that requests both position and rotation

**Comment by salinacsoto** [comment]
I have tested this and this is QA Approved.

In Desktop: 
The animation should look realistic with avatar arm outstretched and flamethrower in hand pointing forward. - PASS
Check that both hands have the correct animation and gun position. - PASS

**Pull request merged**